explainer.md

```diff
@@ -1,624 +1,397 @@
+
+However, these variations could be included in `element()` using modifiers or similarly named functions (e.g. `element-children()`).
 
 # Security/Privacy Considerations
 
-The security considerations below cover same-origin transitions. These are a
-subset of what's required for cross-origin transitions :
-
-*   Script can never read pixel content for images generated using the element()
-    function. This is necessary since the document may embed cross-origin
-    content (iframes, CORS resources, etc.) and multiple restricted user
-    information (visited links history, dictionary used for spell check, etc.)
-*   The Live Animatable Properties could reference resources which are
-    restricted in the new document for an MPA navigation. For example, the old
-    page may use a cross-origin image for border-image which can't be accessed
-    by the new page due to differences in
-    [COEP](https://wicg.github.io/cross-origin-embedder-policy/). Fetching these
-    styles will fail on the new page. For same-origin navigations, the developer
-    already has knowledge of the cross-origin policy on the new page. They can
-    ensure not to reference cross-origin resources in the properties made live.
-
-# Related Reading
-
-An aspect of the feature that needs to be defined is the
-[type of navigations](https://github.com/WICG/app-history#appendix-types-of-navigations)
-that the old page can configure. We expect this will closely align with the
-navigations that can be observed by the page using app-history's
-[navigate event](https://github.com/WICG/app-history#restrictions-on-firing-canceling-and-responding).
-
-[^1]: The pseudo elements in the top layer will not have an associated
-    [::backdrop](https://fullscreen.spec.whatwg.org/#::backdrop-pseudo-element)
-    that is created for other elements in the top layer.
-[^2]: There is a double draw of the shared element in the demo since it
-    continues to paint in the original DOM/the snapshot of the root element.
-    This is addressed in the proposed modifications to the element() function.
+The security considerations below cover same-origin transitions.
+
+* Script can never read pixel content in the textures. This is necessary since the document may embed cross-origin content (iframes, CORS resources, etc.) and multiple restricted user information (visited links history, dictionary used for spell check, etc.)
+* If an element is captured 'as a container + child texture', any external resources specified on the container, such as background images, will be re-fetched in the context of the new page to account for differences in sandboxing.
+
+Cross-origin transitions aren't yet defined, but are likely to be heavily restricted.
+
+# Interactivity and accessibility
+
+Page transitions are a purely visual affordance. In terms of interactivity, transition elements will behave like `div`s regardless of the original element.
```

Comment:
Should hit-testing ignore the transition elements? I think this is a good open question. It's not clear to me how input routing should work while we're displaying the transition setup. Since the only content that's visible onscreen is the transition elements and developers can't register event handlers on them.

Reply:
Yeah, the right answer isn't clear to me